.github/workflows/tempo-ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: Tempo Tests
```

Comment:
Let's name this something like "third party" ? Following the example of https://github.com/mmcloughlin/avo/blob/master/.github/workflows/thirdparty.yml